core/primitives-core/src/config.rs

```diff
@@ -321,34 +318,75 @@ pub struct ExtCostsConfig {
     // #############
     /// Base cost for multiexp
     #[cfg(feature = "protocol_feature_alt_bn128")]
+    #[serde(default = "default_alt_bn128_g1_multiexp_base")]
     pub alt_bn128_g1_multiexp_base: Gas,
     /// byte cost for multiexp
     #[cfg(feature = "protocol_feature_alt_bn128")]
+    #[serde(default = "default_alt_bn128_g1_multiexp_byte")]
     pub alt_bn128_g1_multiexp_byte: Gas,
     /// Base cost for sum
     #[cfg(feature = "protocol_feature_alt_bn128")]
+    #[serde(default = "default_alt_bn128_g1_multiexp_sublinear")]
     pub alt_bn128_g1_sum_base: Gas,
     /// byte cost for sum
     #[cfg(feature = "protocol_feature_alt_bn128")]
+    #[serde(default = "default_alt_bn128_pairing_check_base")]
     pub alt_bn128_g1_sum_byte: Gas,
     /// sublinear cost for items
     #[cfg(feature = "protocol_feature_alt_bn128")]
+    #[serde(default = "default_alt_bn128_pairing_check_byte")]
     pub alt_bn128_g1_multiexp_sublinear: Gas,
     /// Base cost for pairing check
     #[cfg(feature = "protocol_feature_alt_bn128")]
+    #[serde(default = "default_alt_bn128_g1_sum_base")]
     pub alt_bn128_pairing_check_base: Gas,
     /// Cost for pairing check per byte
     #[cfg(feature = "protocol_feature_alt_bn128")]
+    #[serde(default = "default_alt_bn128_g1_sum_byte")]
     pub alt_bn128_pairing_check_byte: Gas,
 }
 
 // We multiply the actual computed costs by the fixed factor to ensure we
 // have certain reserve for further gas price variation.
 const SAFETY_MULTIPLIER: u64 = 3;
 
-/// TODO #4649: remove default implementation when LowerEcrecoverBaseCost will be released
-impl Default for ExtCostsConfig {
-    fn default() -> ExtCostsConfig {
+#[cfg(feature = "protocol_feature_alt_bn128")]
+fn default_alt_bn128_g1_multiexp_base() -> Gas {
+    SAFETY_MULTIPLIER * 237668976500
+}
+
+#[cfg(feature = "protocol_feature_alt_bn128")]
+fn default_alt_bn128_g1_multiexp_byte() -> Gas {
+    SAFETY_MULTIPLIER * 1111697487
+}
+
+#[cfg(feature = "protocol_feature_alt_bn128")]
+fn default_alt_bn128_g1_multiexp_sublinear() -> Gas {
+    SAFETY_MULTIPLIER * 1441698
+}
+
+#[cfg(feature = "protocol_feature_alt_bn128")]
+fn default_alt_bn128_pairing_check_base() -> Gas {
+    SAFETY_MULTIPLIER * 3228502967000
+}
+
+#[cfg(feature = "protocol_feature_alt_bn128")]
+fn default_alt_bn128_pairing_check_byte() -> Gas {
+    SAFETY_MULTIPLIER * 8858396182
+}
+
+#[cfg(feature = "protocol_feature_alt_bn128")]
+fn default_alt_bn128_g1_sum_base() -> Gas {
+    SAFETY_MULTIPLIER * 1058438125
+}
+
+#[cfg(feature = "protocol_feature_alt_bn128")]
+fn default_alt_bn128_g1_sum_byte() -> Gas {
+    SAFETY_MULTIPLIER * 25406181
+}
```

Comment:
That feels suboptimal. Can we just include these in json configs? They should be ignored by stable code. 

Also, @bowenwang1996 what's up with alt_bn128 thing at all? Do we plan to eventually stabilze it? Do we want to remove it?

Reply:
Yes we are right now going through an audit of its implementation